recipes-bsp/u-boot/u-boot_%.bbappend

```diff
@@ -37,6 +37,9 @@ OMNECT_BOOTLOADER_CHECKSUM_FILES += "${OMNECT_THISDIR_SAVED}/u-boot-scr.bb"
 OMNECT_BOOTLOADER_CHECKSUM_FILES += "${OMNECT_THISDIR_SAVED}/u-boot/*"
 
 OMNECT_BOOTLOADER_CHECKSUM_FILES_GLOB_IGNORE = "${OMNECT_THISDIR_SAVED}/u-boot/.gitignore"
+# don't include files which are generated on build:
+OMNECT_BOOTLOADER_CHECKSUM_FILES_GLOB_IGNORE += "${OMNECT_THISDIR_SAVED}/u-boot/redundant-env-fragment.cfg"
```

Comment:
other option would be, we refactor so that redundant-env-fragment.cfg is generated in `WORKDIR`

Reply:
either way is fine for me; if we faced lots of such exceptions in future maybe having that stuff in WORKDIR would be preferable